cmsl1t/utils/fit_efficiency.py

```diff
@@ -30,6 +30,9 @@ def fit_efficiency(efficiency_graph, in_mean, in_sigma=10,
     x_min = efficiency_graph.lowerbound(0)
     x_max = efficiency_graph.upperbound(0)
 
+    if isinstance(in_mean, str):
+        in_mean = 50
```

Comment:
No, it's just a reasonable number in most instances.  I'll add a comment to explain why this is necessary and why I choose 50.